platform_plugin_turnitin/models.py

```diff
@@ -1,3 +1,16 @@
 """
 Database models for platform_plugin_turnitin.
 """
+
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class TurnitinSubmission(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="submissions")
```

Comment:
I think `submissions` is too broad. What about `turnitin_submissions` for the related_name?